synth-publish.go

```diff
@@ -220,8 +221,41 @@ func checkPublishingStatus(latest postedData, result chan<- publicationResult, s
 
 	switch resp.StatusCode {
 	case http.StatusOK:
+		cmdR := exec.Command("kubectl", "delete", "cm", "synthetic-image-alarm", "--ignore-not-found")
+		cmdI := exec.Command("kubectl", "delete", "pods", "--selector=job-name=image-trace-job", "--ignore-not-found=true")
+
+		errR := cmdR.Run()
+		if errR != nil {
+			log.Fatalf("cmdR.Run() failed with %s\n", errR)
+		}
+
+		errI := cmdI.Run()
+		if errI != nil {
+			log.Fatalf("cmdR.Run() failed with %s\n", errI)
+		}
 	case http.StatusNotFound:
 		handlePublishingErr(result, latest.tid, latest.time, "Image not found. Response status code: 404.")
+		cmd1 := exec.Command("kubectl", "delete", "jobs", "image-trace-job", "--ignore-not-found")
+		cmd2 := exec.Command("kubectl", "delete", "configmap", "synthetic-tid")
+		cmd3 := exec.Command("kubectl", "create", "configmap", "synthetic-tid", "--from-literal=TID="+latest.tid)
```

Comment:
`if err := k8sNotFound(); err != nil {
	log.Fatal(err.Error())
}`

Reply:
There is "latest.tid" here which is only valid in checkPublishingStatus function so I've decided to not use such function